src/chia_log/handlers/daily_stats/stats_manager.py

```diff
@@ -96,7 +103,10 @@ def _run_loop(self):
         while self._is_running:
```

Comment:
L93 reads 'here's what happened in the last 24 hours:'
This is incorrect when the frequency_hours is not set to 24. 
Debug mode should also be handled here.

Reply:
Good catch